iguazio/prestosql
# Upgrade to prestosql 326 (#7)

* Update documentation for SELECT

* Implement generalized .*

* Introduce IcebergPageSource and IcebergColumnHandle

This commit simplifies logic and fixing bugs in case of Iceberg
Partition Spec evolution.

* Test asterisked identifier chain analysis

* Document default max rows in Google Sheets Connector

* Fix Unicode rendering in CLI pager in Docker image

* Clarify Block#getChildren semantics

* Remove unnecessary predication pushdown

The parameter constraint in PartitionTable#cursor is targetted at
columns in the final partition table, rather than the original
data table.

* Test select all with order by and aggregation

* Fix Parquet Predicate Pushdown on Dictionaries

Commit 0f7982b refactored ParquetPredicateUtils.getDictionaries from
getDictionariesByColumnOrdinal, removing a nested loop iteration but
accidentally leaving in a break statement. The effect has been that
at most 1 dictionary was returned from getDictionaries, limiting the
effectiveness of predicate pushdown on dictionaries.

* Add support for datetimes exposed as numbers in Elasticsearch

Older version of Elasticsearch use epoch millis instead of ISO-8601
when formatting datetime fields in document fields output.

* Stop Parquet Dictionary Reads Eagerly in Predicate Pushdown

No more parquet dictionaries need to be read once a dictionary
predicate pushdown check succeeds.

* Add 323 release notes

* [maven-release-plugin] prepare release 323

* [maven-release-plugin] prepare for next development iteration

* Run two hives product tests with non generic suite

* Fix type inference for simple case expression operand

* Merge release 323 to dev (#6)

* Update documentation for SELECT

* Implement generalized .*

* Introduce IcebergPageSource and IcebergColumnHandle

This commit simplifies logic and fixing bugs in case of Iceberg
Partition Spec evolution.

* Test asterisked identifier chain analysis

* Document default max rows in Google Sheets Connector

* Fix Unicode rendering in CLI pager in Docker image

* Clarify Block#getChildren semantics

* Remove unnecessary predication pushdown

The parameter constraint in PartitionTable#cursor is targetted at
columns in the final partition table, rather than the original
data table.

* Test select all with order by and aggregation

* Fix Parquet Predicate Pushdown on Dictionaries

Commit 0f7982b refactored ParquetPredicateUtils.getDictionaries from
getDictionariesByColumnOrdinal, removing a nested loop iteration but
accidentally leaving in a break statement. The effect has been that
at most 1 dictionary was returned from getDictionaries, limiting the
effectiveness of predicate pushdown on dictionaries.

* Add support for datetimes exposed as numbers in Elasticsearch

Older version of Elasticsearch use epoch millis instead of ISO-8601
when formatting datetime fields in document fields output.

* Stop Parquet Dictionary Reads Eagerly in Predicate Pushdown

No more parquet dictionaries need to be read once a dictionary
predicate pushdown check succeeds.

* Add 323 release notes

* [maven-release-plugin] prepare release 323

* [maven-release-plugin] prepare for next development iteration

* Run two hives product tests with non generic suite

* Fix type inference for simple case expression operand

* Support V3IO  authentication token and parser in PrestoSQL Hive module

* Fix authentication issue (#5)

* Support V3IO  authentication token and parser in PrestoSQL Hive module

* Implement V3IO dynamic configuration provider

* addressing PR comments

* Merge PrestoSQL 326 tag
